### ISSUE TITLE
User objects sometimes not storing avatar

### ISSUE THREAD

**Issue by hydrabolt** [hydrabolt/discord.js#32]
As reported by some users and having experienced it myself, it's pretty annoying.

The User object sometimes does not give an avatar, and a simple relog can fix this. I'm having a look at it.

**Comment by murm** [comment]
More information:

```User {
  username: undefined,
  discriminator: undefined,
  id: '85324585536327680',
  avatar: undefined,
  status: 'online' }```

That is message.author logged when user is considered undefined.

**Comment by hydrabolt** [comment]
Ok this makes me think that the update to the new API meant that only the ID was sent, will look into it some more, thanks!

**Comment by hydrabolt** [comment]
Ok, I think I've fixed it, have a look at 083ed542744831c27e1ee3c8cf1b1d7aa627de31. Thanks! Re-open if it reoccurs
